spec/integration/sidekiq7_spec.rb

```diff
@@ -0,0 +1,193 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Sidekiq client logger' do
+  let(:target) { StringIO.new }
+  let(:config) { Sidekiq.instance_variable_get :@config }
```

Comment:
`Sidekiq.default_configuration` should be the same https://github.com/sidekiq/sidekiq/blob/v7.0.7/lib/sidekiq.rb#L89